violet-framework/src/main/java/com/horstmann/violet/workspace/spellchecker/SpellChecker.java

```diff
@@ -0,0 +1,36 @@
+package com.horstmann.violet.workspace.spellchecker;
+
+import java.io.IOException;
+import java.util.List;
+import org.languagetool.JLanguageTool;
+import org.languagetool.language.BritishEnglish;
+import org.languagetool.rules.Rule;
+import org.languagetool.rules.RuleMatch;
+
+/**
+ * This class check correctness of string
+ *
+ * @author Sebastian Podgorski
```

Comment:
I think it isn't the only one class with @author annotation, so please check all classes for this annotation